test/controllers/v1/profiles_controller_test.rb

```diff
@@ -11,6 +11,29 @@ class ProfilesControllerTest < ActionDispatch::IntegrationTest
       profiles(:one).update! account: accounts(:test)
     end
 
+    context 'index' do
+      should 'not contain external or canonical profiles by default' do
+        get v1_profiles_url
+        assert_response :success
+
+        profiles = JSON.parse(response.body)
+        assert_empty profiles['data']
+      end
+
+      should 'only contain internal profiles by default' do
```

Comment:
Why are we using `should` here? It doesn't match any of our other controller tests.

Reply:
We do use should from and context from shoulda-context in `test/controllers/concerns/metadata_test.rb`, `test/controllers/concerns/authentication_test.rb` and many service and model tests

Reply:
Ok, it's not really a big deal, just weird to have mixed syntax in a single test file.